mathesar/models/base.py

```diff
@@ -121,3 +121,21 @@ class Meta:
                 name="frac_digits_integrity"
             )
         ]
+
+
+class TableMetaData(BaseModel):
+    database = models.ForeignKey('Database', on_delete=models.CASCADE)
+    schema_oid = models.PositiveBigIntegerField()
+    table_oid = models.PositiveBigIntegerField()
+    import_verified = models.BooleanField(default=False)
+    column_order = models.JSONField(default=list)
+    preview_customized = models.BooleanField(default=False)
+    preview_template = models.CharField(max_length=255, blank=True)
```

Comment:
1. I'd like these fields to be renamed:

    - `preview_customized` → `record_summary_customized`
    - `preview_template` → `record_summary_template`

    These changes will serve to make our terminology more consistent across the stack.

1. As a side note: I'm not sure we need `preview_customized`. I did a little digging on this and couldn't figure out why it's necessary. Perhaps we _do_ need it. I don't think we ought to try removing it in this PR, but I just want to mention this as a heads up. I might want to remove this later once we get deeper into the record summary work before beta. If anyone seeing this comment happens to have a clear understanding of why we need this field, please comment here. A code comment explaining it would be nice too! But not worth spending much time clarifying right now.